releasenotes/releasenotes_develop.md

```diff
@@ -1,52 +1,299 @@
+- Auto-updating go-utils and kubernetes-utils in keptn/keptn needs to be a signed commit (and moved to GitHub Actions) [2750](https://github.com/keptn/keptn/issues/2750)
+
+</p>
+</details>
 
+<details><summary>Miscellaneous CI tasks (for build, test, quality checks)</summary>
+<p>
+
+- Multi-architecture build support for CLI (32 bit, ARM, ...) [2997](https://github.com/keptn/keptn/issues/2997)
+- Add dependabot to keep dependencies up2date [2648](https://github.com/keptn/keptn/issues/2648)
+- Switch from CLA Bot to DCO [2690](https://github.com/keptn/keptn/issues/2690)
+- Solved problems with test coverage reporting [2929](https://github.com/keptn/keptn/issues/2929)
+- Various improvements for GH actions [2824](https://github.com/keptn/keptn/issues/2824)
+- Include pluto to automatically check for deprecated K8s apiVersions [2382](https://github.com/keptn/keptn/issues/2383)
+- Integration tests: enable shielded GKE nodes for integration tests and in docs [2973](https://github.com/keptn/keptn/issues/2973)
+- Integration tests: use newer Istio version [2976](https://github.com/keptn/keptn/issues/2976)
+- Integration tests (GKE for 1.16): Legacy monitoring is not supported in this version [2789](https://github.com/keptn/keptn/issues/2789)
+- After a feature/bug/patch/hotfix has been merged, the respective (temporary) images are deleted [1037](https://github.com/keptn/keptn/issues/1037)
+- DockerHub: Stale images are going to be deleted soon [2710](https://github.com/keptn/keptn/issues/2710)
+- Move tests for delivery assistant and self-healing to K3s [2771](https://github.com/keptn/keptn/issues/2771)
+- Reduce the number of platform/integration tests on Travis-CI [2718](https://github.com/keptn/keptn/issues/2718)
+- Makefile: *Fixed* - Build-CLI works, but the resulting binary is not [2504](https://github.com/keptn/keptn/issues/2504)
+- Makefile: Add a way to build all Dockerfile [2464](https://github.com/keptn/keptn/issues/2464)
+- Makefile: Add build and run targets [2405](https://github.com/keptn/keptn/issues/2405)
+
+</p>
+</details>
+
+<details><summary>Fixed CI issues</summary>
+<p>
+
+- *Fixed*: Integration tests: GKE clusters are not deleted afterwards in some cases [3243](https://github.com/keptn/keptn/issues/3243)
+- *Fixed*: Flaky integration tests: Integration tests fail (in unpredictable situations) [2149](https://github.com/keptn/keptn/issues/2149)
+- *Fixed*: Integration test stalls at the Keptn auth command [2704](https://github.com/keptn/keptn/issues/2704)
+- *Fixed*: Integration test: Setup of Keptn fails due to server version check [2701](https://github.com/keptn/keptn/issues/2701)
+- *Fixed*: Unable to do remote debugging of mongodb-datastore due to liveness-probe [2536](https://github.com/keptn/keptn/issues/2536)
+- *Fixed*: GitHub Action Reviewdog Fails: The `add-path` command is disabled [2694](https://github.com/keptn/keptn/issues/2694)
+
+</p>
+</details>
+
+## Good to know / Known Limitations
```

Comment:
Some known limitations:
* https://github.com/keptn/keptn/issues/3208
* https://github.com/keptn/keptn/issues/2989 (though here I'm not sure, see comment on the issue)
* https://github.com/keptn/keptn/issues/2453
* https://github.com/keptn/keptn/issues/2369 (copied from 0.7.3)

Reply:
Thanks for listing the know limitations. I added them.